docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -255,6 +255,89 @@ Standard TensorFlow\* operations:
 | ZerosLike | No |
 
 
+## TensorFlow 2 Keras\* Supported Operations
+
+Standard TensorFlow 2 Keras\* operations:
+
+| Operation Name in TensorFlow 2 Keras\* | Limitations|
+| :----------| :----------|
+| ActivityRegularization | No |
+| Add | No |
+| AdditiveAttention | No |
+| AlphaDropout | No |
+| Attention | No |
+| Average | No |
+| AveragePooling1D | No |
+| AveragePooling2D | No |
+| AveragePooling3D | No |
+| BatchNormalization | No |
+| Bidirectional | No |
+| Concatenate | No |
+| Conv1D | No |
+| Conv1DTranspose | Configuration with dilation not equal to 1 is not supported |
+| Conv2D | No |
+| Conv2DTranspose | No |
+| Conv3D | No |
+| Conv3DTranspose | No |
+| Cropping1D | No |
+| Cropping2D | No |
+| Cropping3D | No |
+| Dense | No |
+| DenseFeatures | The layer is not fully supported and pruning is required |
```

Comment:
`layer` -> `operation`

Reply:
It is not clear what does "pruning is required"? Pruning when and where?

Reply:
I clarify what cases are not supported. Pruning is up to user and a different topic.